metrics.md

```diff
@@ -221,11 +223,11 @@ In the metrics below, we adopt the following nomenclature:
 
 ### Brier Score (BS)
 {: .anchor }
-The BS measures the accuracy of forecast probability for one or more events:
+The BS measures the accuracy of forecast probability for one or more events ([Brier50](#ref-brier50)). For single event (binary) outcomes, BS is defined as:
 
 $$ \text{BS} = \frac{1}{n} \sum_{i=1}^n (f_i - o_i)^2  $$
 
-Smaller values of BS indicate better agreement between forecasts and observations.
+Smaller values of BS indicate better agreement between forecasts and observations. Note that while BS can be generalized to multi-categorical events, the Solar Forecast Arbiter only includes built-in support for the (more commonly used) single event BS (as defined above).
```

Comment:
Similar to the above, 'events with more than two outcomes' rather than 'multi-categorical events' and …. 'binary events'  (rather than 'single event')

Reply:
Revising to "...support for the (more commonly used) binary events definition."